var/spack/repos/builtin/packages/ispc/package.py

```diff
@@ -67,10 +67,15 @@ class Ispc(CMakePackage):
         sha256="d3ccf547d3ba59779fd375e10417a436318f2200d160febb9f830a26f0daefdc",
     )
 
+    # Fix library lookup for NCurses in CMake
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/ispc/ispc/pull/2638.patch?full_index=1",
+        when="@1.18:1.20",
+        sha256="3f7dae8d4a683fca2a6157bbcb7cbe9692ff2094b0f4afaf29be121c02b0b3ad",
+    )
+
     def setup_build_environment(self, env):
         if self.spec.satisfies("@1.18.0:"):
-            env.append_flags("LDFLAGS", "-lcurses")
-            env.append_flags("LDFLAGS", "-ltinfo")
```

Comment:
Patch doesn't add `tinfo` library, only `curses`. Can you comment on its need?

Reply:
What you don't see in that patch is the tinfo is included in the `CURSES_LIBRARIES` if it is enabled by ncurses.

Reply:
Does this then need `depends_on("ncurses +termlib")` to make sure we actually have `tinfo`?

Reply:
(at least, I think libtinfo is only there when `+termlib`)

Reply:
This was there in order to resolve symbols required by LLVM libraries which are linked into ispc. There was no good way of finding out what LLVM requires, hence this was hard coded, I think. If Curses does not provide libtinfo, then it should not be required by LLVM either.